Dockerfile

```diff
@@ -44,12 +43,16 @@ COPY templates /var/www/templates
 COPY config /var/www/config
 COPY src /var/www/src
 COPY data /var/www/data
+COPY public /var/www/public
+
+# Copy assets from docker assets stage in a not mapped folder to be maintained
+RUN mkdir -p /assets && mkdir -p /assets/css
+COPY --from=assets /assets/css/*.css /assets/css/
 
 # Reset "local"/development config files
 RUN rm -f /var/www/config/development.config.php && \
-  rm /var/www/config/autoload/*.local.php && \
-  mv /var/www/config/autoload/local.php.dist /var/www/config/autoload/local.php
+    rm -f /var/www/config/autoload/*.local.php && \
+    mv /var/www/config/autoload/local.php.dist /var/www/config/autoload/local.php
 
 # Build project
 WORKDIR /var/www
-RUN composer install --quiet --no-ansi --no-dev --no-interaction --no-progress --no-scripts --no-plugins --optimize-autoloader
```

Comment:
Why is this moved to the entrypoint?

In production, this is no good, as it means it takes longer to deploy (we now have to wait for composer to install before the server can start).

Reply:
This was also done because the volume mapping lost the files generated by `composer install` with docker-compose.yml

Maybe there's another better alternative.

Reply:
Okay, I think I understand what's going on here finally, and it's a difference in development patterns.

When I'm developing locally, I'm running `composer` using a local PHP binary, so the `vendor` directory is present and usable at that point, as the application root directory is mapped to the `/var/www` directory in the container.

My guess is you're just firing up `docker-compose up` without doing a `composer install` first (or even later), which then means the entire application breaks for you.

I'm going to give the container a try within a bare checkout, and then see if the changes you propose fix them.

Reply:
I have an idea on how to solve this; if it works, I'll push the changes to your branch.